simtools/applications/calculate_trigger_rate.py

```diff
@@ -97,11 +97,63 @@ def _parse(label, description):
         action="store_true",
     )
 
+    config.parser.add_argument(
+        "--energy_range",
+        help="Energy range (TeV) passed as a list of floats, the minimum and maximum, "
+        "respectively.",
+        required=False,
+        nargs=2,
+        type=str,
+        default=None,
+    )
+
+    config.parser.add_argument(
+        "--view_cone",
+        help="View cone (deg) passed as two floats, the minimum and maximum, respectively.",
```

Comment:
Maybe 'View cone radius (deg)' (I assume it is the radius, or?)

Reply:
True. I adjusted the text.